articles/architecture-scenarios/implementations/spa-api/spa-implementation-angular2.md

```diff
@@ -472,7 +472,11 @@ And add a file called `silent.html`:
 <html>
 <head>
   <meta charset="utf-8">
+<<<<<<< HEAD
   <script src="${auth0js_url}"></script>
+=======
+  <script src="${auth0js_urlv9}"></script>
+>>>>>>> Version URL updates
```

Comment:
Please fix this. This example uses the silent url callback page so it needs to be updated